### PR TITLE
Add es translation

### LINKED ISSUE #53: Add i18n support
- [ ] Add another `mkdocs.yml` for the other language
- [ ] Configure  [markdown-i18n](https://github.com/gisce/markdown-i18n)
- [ ] Add travis tests to check all documentation is translated
  - Check if `locales/lang/LC_MESSAGES/messages.po` is completly translated (every msg.id has a msg.string)
  - Check all the strings in `locales/messages.pot`  are in every `locales/lang/LC_MESSAGES/messages.po`
  - The build url should use `lang` as http://builds.gisce.net/powerp-docs/es/powerp_new_cups_sync/

### PR DESCRIPTION
**Pull request by jaumef**
Closes #53 

Added default directories

Update:

- [x] Readme
- [x] mkdocs.yml
- [x] travis tests

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL: http://builds.gisce.net/powerp-docs/powerp_add_es_translation/

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 mkdocs
+markdown_i18n
```
Comment (ecarreras) [inline requirements.txt]: the package is `markdown-i18n`

Comment (ecarreras) [inline requirements.txt]: Yes, but is different the name of the package for installing from pip which is `markdown-i18n` and the extension name which is the python module `markdown_i18n`

**Comment by giscegit** [comment]
Documentation build URL: http://builds.gisce.net/powerp-docs/powerp_add_es_translation/

**Comment by ecarreras** [comment]
image

Check you should fix sitecustomize to use UTF-8

**Comment by giscegit** [comment]
Documentation build URL: http://builds.gisce.net/powerp-docs/powerp_add_es_translation/

**Comment by giscegit** [comment]
Documentation build URL: http://builds.gisce.net/powerp-docs/powerp_add_es_translation/

**Comment by giscegit** [comment]
Documentation build URL: http://builds.gisce.net/powerp-docs/powerp_add_es_translation/

**Comment by giscegit** [comment]
Documentation build URL: http://builds.gisce.net/powerp-docs/powerp_add_es_translation/

**Comment by giscegit** [comment]
Documentation build URL: http://builds.gisce.net/powerp-docs/powerp_add_es_translation/

**Pull request merged**